addons/website_slides/models/slide_channel.py

```diff
@@ -518,13 +523,13 @@ def message_post(self, *, parent_id=False, subtype_id=False, **kwargs):
     # ---------------------------------------------------------
 
     def action_redirect_to_members(self, state=None):
+        self.ensure_one()
         action = self.env["ir.actions.actions"]._for_xml_id("website_slides.slide_channel_partner_action")
-        action['domain'] = [('channel_id', 'in', self.ids)]
-        if len(self) == 1:
-            action['display_name'] = _('Attendees of %s', self.name)
-            action['context'] = {'active_test': False, 'default_channel_id': self.id}
-        if state:
-            action['domain'] += [('completed', '=', state == 'completed')]
+        action['display_name'] = _('Attendees of %s', self.name)
+        action_ctx = {'search_default_channel_id': self.ids, 'active_test': False}
+        if state and state == 'completed':
```

Comment:
This method is also called by "action_redirect_to_running_members" but it seems unused.

I think you should take this opportunity to remove the "action_redirect_to_running_members" method as well, to do some cleaning.

And you can change this method to receive "completed=False" instead of a "state=None", to make the API a little more simple.
Then "action_redirect_to_done_members" will call it passing completed=True.

Reply:
@aaj-odoo I think you trashed @dja-odoo modifications when force-pushing.

I remember he had done this change and now it seems gone?